apps/condo/domains/ticket/components/IncidentHints.tsx

```diff
@@ -46,6 +46,12 @@ const IncidentHint: React.FC<IncidentHintProps> = (props) => {
 
     const { incident } = props
 
+    const trimmedDetails = useMemo(() => {
+        const details = get(incident, 'details', '')
+        return details.length > MAX_DETAILS_LENGTH ? `${details.substring(0, MAX_DETAILS_LENGTH)}…` : details
+    }, [incident])
```

Comment:
this code depends on `details.length` too

Reply:
Trimming text by length does not take into account block width, font size, etc. 
Are you sure it's the best solution?

Reply:
This is the designers decision, not mine

code depends on `incident` only